kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -92,26 +93,26 @@ else
             --build-arg environment=k8s-local \
             --build-arg target="${arch/#arm/aarch}"-unknown-linux-gnu \
             ../../ \
-            -t "$docker_image" || exit 1
+            -t "$docker_image"
     fi
 fi
 
-kind load docker-image "$docker_image" || exit 1
+kind load docker-image "$docker_image"
 
-helm uninstall linera-core --wait;
+helm uninstall linera-core --wait || true
 
 if [ -n "$cloud_mode" ]; then
-    helm install linera-core . --values values-local-with-cloud-build.yaml --wait --set installCRDs=true || exit 1;
+    helm install linera-core . --values values-local-with-cloud-build.yaml --wait --set installCRDs=true
 else
-    helm install linera-core . --values values-local.yaml --wait --set installCRDs=true || exit 1;
+    helm install linera-core . --values values-local.yaml --wait --set installCRDs=true
 fi
 
 echo "Pods:"
 kubectl get pods
 echo -e "\nServices:"
 kubectl get svc
 
-docker rm linera-test-local
+docker rm linera-test-local || true
 docker run -d --name linera-test-local "$docker_image" &&
     docker cp linera-test-local:wallet.json /tmp/ &&
     docker cp linera-test-local:linera.db /tmp/
```

Comment:
```suggestion
docker run -d --name linera-test-local "$docker_image"
docker cp linera-test-local:wallet.json /tmp/
docker cp linera-test-local:linera.db /tmp/
```

Reply:
I think this was the statement that originally prompted this PR!  Let's switch this too and then I'm happy :)